sdk/examples/wallet/ledger_nano.rs

```diff
@@ -17,64 +24,78 @@ use iota_sdk::{
     wallet::{ClientOptions, Result, SendAmountParams, Wallet},
 };
 
+// The account alias used in this example
+const ACCOUNT_ALIAS: &str = "ledger";
+// The wallet database folder created in this example
+const WALLET_DB_PATH: &str = "./example.ledger_nano.walletdb";
+// The number of addresses to generate
+const NUM_ADDRESSES_TO_GENERATE: u32 = 1;
+// The address to send coins to
+const RECV_ADDRESS: &str = "rms1qpszqzadsym6wpppd6z037dvlejmjuke7s24hm95s9fg9vpua7vluaw60xu";
+// The amount of base coins we'll send
+const SEND_AMOUNT: u64 = 1_000_000;
+
 #[tokio::main]
 async fn main() -> Result<()> {
     // This example uses secrets in environment variables for simplicity which should not be done in production.
     dotenvy::dotenv().ok();
 
-    let client_options = ClientOptions::new().with_node(&std::env::var("NODE_URL").unwrap())?;
-
+    let client_options = ClientOptions::new().with_node(&var("NODE_URL").unwrap())?;
     let secret_manager = LedgerSecretManager::new(true);
-
     let wallet = Wallet::builder()
         .with_secret_manager(SecretManager::LedgerNano(secret_manager))
-        .with_storage_path("ledger_nano_walletdb")
+        .with_storage_path(WALLET_DB_PATH)
```

Comment:
i think we can use the normal wallet path here as well

Reply:
Done